### ISSUE TITLE
[BUG]: --detach no longer working with --type=channel

### ISSUE THREAD

**Issue by 2084x** [pystardust/ytfzf#594]
### Describe the bug
when using '--type=channel' alongside '--detach', selected video does not detach.
### To Reproduce
run ytfzf --type=channel --detach -l channelname

select a channel, then select a video. video will not detach and will instead be swallowed by the terminal.
### Expected behavior
run ytfzf --type=channel --detach -l channelname

select a channel, then select a video. video will detach and original window will stay as it is, allowing you to select and detach other videos from the same channel.

### Information
 - OS: arch linux
 - Terminal: st
 - Ytfzf version: 2.5.2
- Output of `ls -l "$(which sh)"` (if you're using fish: `ls -l (which sh)`): bash (I use zsh though)

### Additional context
The reason I do this is because swallowing and looping makes me lose thumbnails when using dwm/st. If I detach I can keep the thumbnails and select as many videos as I want without running another search. Previously this was working fine. Was something changed in the last update?  '--detach' is still working with '--type=all' and '-c youtube-subscriptions', (those are some I have aliased so I tested them) so it just seems like a problem with the channel search.

**Comment by Euro20179** [comment]
TLDR: use `--submenu-opts=--detach` instead of `--detach`

The reason this does not work is because selecting a channel opens a submenu, opening a submenu basically resets all options (there are a few that are not reset, such as your interface, and url handler), however the rest are reset as it is impossible to know what should and should not be passed through.

**Comment by 2084x** [comment]
Ah sweet. I didn't realise submenus were a thing... I see there's also a specific conf file for it so I can just use that. Thankyou.
